metadata.rb

```diff
@@ -9,6 +9,7 @@
 version          '1.3.1'
 
 depends          'line'
+depends          'osl-selinux'
```

Comment:
I wanted to only depend on public cookbooks for this wrapper cookbook to keep the depencies cleaner and more open. Can you please change this just to the `selinux` cookbook?

Reply:
Done!